Source/WebCore/accessibility/AXObjectCache.cpp

```diff
@@ -1368,7 +1357,7 @@ void AXObjectCache::handleChildrenChanged(AccessibilityObject& object)
             shouldUpdateParent = false;
         }
 
-        if (parent->isLabel()) {
+        if (parent->isNativeLabel()) {
```

Comment:
Current `main` behavior runs this code when:

`bool isLabel() const { return isAccessibilityLabelInstance() || labelForObjects().size(); }`

Don't we need to run this branch when `isLabelByRelation()` as well to maintain the existing behavior? I'd assume some tests would start failing when we only do this for `isNativeLabel()` 🤔